lib/src/common/model/repos_releases.dart

```diff
@@ -42,10 +53,12 @@ class Release {
   String description;
 
   /// If the release is a draft.
-  bool draft;
+  @JsonKey(name: "draft")
+  bool isDraft;
 
```

Comment:
Could you add a deprecated getter with the original name so this is a non-breaking change?
Like this?
```suggestion

@deprecated
bool get draft => isDraft;
```

Reply:
What about the json key?

Reply:
The json key can stay with isDraft. I believe JSON deserialization will be ok with that.
All of these model classes are part of the public API. Consumers could be getting a Release object and checking the draft property. Renaming it means it would require a change in consumer code to work, which makes it a breaking change.